subprojects/scala/src/integTest/groovy/org/gradle/scala/ScalaCompilationFixture.groovy

```diff
@@ -14,23 +14,38 @@
  * limitations under the License.
  */
 
-package org.gradle.scala.compile
+package org.gradle.scala
 
+import org.gradle.integtests.fixtures.ScalaCoverage
+import org.gradle.language.scala.internal.toolchain.DefaultScalaToolProvider
 import org.gradle.test.fixtures.file.TestFile
 
-class ScalaIncrementalCompilationFixture {
+class ScalaCompilationFixture {
     private final TestFile root
     final ScalaClass basicClassSource
     final ScalaClass classDependingOnBasicClassSource
     final ScalaClass independentClassSource
     final String sourceSet
+    String scalaVersion
+    String zincVersion
+    String sourceCompatibility
+    String sourceDir
 
-    ScalaIncrementalCompilationFixture(File root) {
+    ScalaCompilationFixture(File root) {
         this.root = new TestFile(root)
         this.sourceSet = 'main'
+        this.sourceDir = 'src/main/scala'
+        this.scalaVersion = ScalaCoverage.NEWEST
+        this.zincVersion = DefaultScalaToolProvider.DEFAULT_ZINC_VERSION
+        this.sourceCompatibility = '1.7'
         basicClassSource = new ScalaClass(
             'Person',
-            'class Person(val name: String, val age: Int)',
+            '''
+                /**
+                 * A person.
+                 * Can live in a house.
+                 */
+                class Person(val name: String, val age: Int)'''.stripIndent(),
             'class Person(val name: String, val age: Int, val height: Int)')
```

Comment:
:o: It would be nice to keep these symmetric, i.e. add the Javadoc to the changed code as well.